src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -183,4 +184,12 @@ public static string Tcp_CustomBinding_SslStreamSecurity_Address
             return BridgeClient.GetResourceAddress("WcfService.TestResources.TcpTransportSecurityWithSslResource");
         }
     }
+
+    public static string Tcp_CustomBinding_SslStreamSecurity_HostName
+    {
+        get
+        {
+            return BridgeClient.GetResourceHostName("WcfService.TestResources.TcpTransportSecurityWithSslResource");
+        }
```

Comment:
We should have a chat about resource naming.  I would prefer we are consistent in using "Address" or "Resource" as the suffix and making the name of the static string match the name of the resource on the server.  Having 2 Endpoints with unique names that point to the same place is confusing to me (as well as having neither match the type name).  NIT -- not a blocker.